build.gradle

```diff
@@ -25,6 +25,7 @@ archivesBaseName = project.name.toLowerCase()
 
 repositories {
     jcenter()
+    mavenCentral()
```

Comment:
This is probably not needed, and if it is it would break fetching the plugin from the Gradle plugin portal (since that only fetches dependencies from `jcenter()`). Most (if not all) dependencies on Maven central are also on `jcenter()` though.